src/main/java/duke/storage/Stats.java

```diff
@@ -0,0 +1,69 @@
+package duke.storage;
```

Comment:
why is stats inside storage? I think it would be better if stats is implemented in commands